docs/telemetry.rst

```diff
@@ -134,3 +134,13 @@ Supported telemetry parameters:
 
 * ``ccr-stats-indices`` (default: all indices): An index pattern for which ccr stats should be checked.
 * ``ccr-stats-sample-interval`` (default 1): A positive number greater than zero denoting the sampling interval in seconds.
+
+transform-stats
```

Comment:
we may also want to add `transform-stats` under `Available telemetry devices:` section above, line 27